less/reactions.less

```diff
@@ -32,7 +32,7 @@
         cursor: pointer;
 
         &.max-reactions {
-            display: none;
+            display: none !important;
```

Comment:
3.0 no longer uses less files so this file should be renamed to scss/reactions.scss and plugin.json should be updated to use the scss block

Reply:
Hmmm... I don't know SCSS syntax, I converted with [this tool](https://jsonformatter.org/less-to-scss), hope it's ok 😄

Reply:
They are mostly indentical, there are some syntax difference related to mixins but they are not used here so you should be good.

Just build nodebb with the pluign active and if there are scss errors they will show up during build.